### PR TITLE
[docs][ybm] Update AWS PSE CLI instructions

### PR DESCRIPTION
**Pull request by ddhodge**
Update AWS CLI instructions for creating an interface endpoint on AWS

@netlify /preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md:1]: Line 36: The Amazon Resource Names (ARN) *or* security principals to which to grant access to the private service endpoint.

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md [hunk 1]
```diff
@@ -121,20 +122,102 @@ To create an interface endpoint to connect to your cluster PSE, do the following
...
+    --service-name <pse_service_name> \
+    --subnet-ids <subnet_ids> \
+    --security-group-ids <endpoint_security_group_id> \
+    -–private-dns-enabled 
 ```
 
 Replace values as follows:
 
 - `application_vpc_id` - ID of the AWS VPC. Find this value on the VPC dashboard in your AWS account.
 - `region` - region where you want the VPC endpoint. The region needs to be the same as a region where your cluster is deployed.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md:215]:
```suggestion
```

**Comment by shinji62** [comment]
@ddhodge Do you me to merge my change into yours ?

**Comment by ddhodge** [comment]
> @ddhodge Do you me to merge my change into yours ?

Not explicitly, already had this PR open so thought you would be interested

**Comment by ddhodge** [comment]
@shinji62 Was on phone misread your comment - yes by all means merge your change into this PR

**Review by mrajcevic01 — APPROVED** [review]

**Pull request merged**